packages/graphql-serve/src/runtime.ts

```diff
@@ -25,19 +28,45 @@ export const createMongoDBClient = async (): Promise<MongoClient> => {
   return client
 }
 
+export function getConflictMapForModels(model: GraphQLSchema, conflictResolutionStrategy: ConflictResolutionStrategy): DataSyncModelConfigMap {
+  const map: DataSyncModelConfigMap = {};
+
+  const dataSyncModelTypes = filterModelTypes(model).filter((type: GraphQLObjectType) => {
+    return !!parseMetadata('datasync', type);
+  })
```

Comment:
It does actually, but it uses ModelDefinition instead of GraphQL Type, hence a bit troublesome to use it here:
https://github.com/aerogear/graphback/blob/bb774b287d6f387964b2cd740775e33b5984dc8e/packages/graphback-datasync/src/util.ts#L5-L7

Reply:
Great. We can add the below function along side this 
- `isDataSyncType(graphqlType)` 

Then the `isDataSyncModel(model: ModelDefinition)` would simply call this function with `model.graphqlType`. 

I guess what I am hoping to see is having this knowledge in one place where it belongs, and not scattered.

Reply:
This entire thing looks like terrible hack for the missing feature (global config) that needs to be added.